test/runtests.jl

```diff
@@ -16,16 +16,15 @@ replace_windows_filepaths(str) = replace(str, ".\\" => "./")
 # Replace Int32s so reference strings work on x86 platforms
 replace_Int32s(str) = replace(str, "Int32" => "Int64")
 
+# remove stacktraces so reference strings work for different Julia versions
+remove_stacktraces(str) = replace(str, r"(Stacktrace:)[^<]*" => "")
+
```

Comment:
shall we add a:
```
const clean_report = replace_Int32s ∘  replace_windows_filepaths  ∘ strip_filepaths ∘  remove_stacktraces
```
and then use that below?

Reply:
I haven't seen function composition before, that's really cool. Why do you use a `const` for it out of interest?